.gitlab/build_tioga.yml

```diff
@@ -17,10 +17,26 @@
     ALLOC_COMMAND: "flux mini run -t10m -n1"
   extends: [.build_script, .on_tioga]
 
+####
+# Template
+.run_project_integration_test_on_tioga:
+  stage: test
+  variables:
+    ALLOC_COMMAND: "flux mini run -t10m -n1"
```

Comment:
```suggestion
    ALLOC_COMMAND: "flux run -t10m -n1"
```

Reply:
This is not resolved. `flux mini` is a deprecated command. All that needs to happen is to remove `mini`.